### ISSUE TITLE
bot stop working

### ISSUE THREAD

**Issue by chelaxian** [joybiswas007/node-midjourney-telegram-bot#6]
bot stop working. telegram-bot is responding, but in discord nothing appears.

**Comment by chelaxian** [comment]
double checked all ID's and tokens - they didn't changed. maybe discord update something and node unofficial client stop working?

**Comment by joybiswas007** [comment]
> double checked all ID's and tokens - they didn't changed. maybe discord update something and node unofficial client stop working?

That's probably it. In that case that unofficial client needs to be updated I guess. Btw just for clarity is your midjourney subscriptions active atm?

**Comment by chelaxian** [comment]
> > double checked all ID's and tokens - they didn't changed. maybe discord update something and node unofficial client stop working?
> 
> That's probably it. In that case that unofficial client needs to be updated I guess. Btw just for clarity is your midjourney subscriptions active atm?

yes, subscrtiption is active and in discord images are generating

**Comment by chelaxian** [comment]
for unofficial node client i meant this api - https://github.com/erictik/midjourney-api
you use it in your project and its was updated 3 days ago.
do you need to update your project also to impleament this update or no?

**Comment by joybiswas007** [comment]
I think I need to update my codes too.

**Comment by chelaxian** [comment]
hi! any updates on code? thx

**Comment by joybiswas007** [comment]
> hi! any updates on code? thx

Hi, I'm so sorry. Unfortunately I don't have active midjourney subscriptions atm so couldn't fix this. Also I check this client's https://github.com/erictik/midjourney-api discord server lots of other peoples are also facing this issue client not working I saw in their discord server. That library needs to be updated.
